fastlane/Fastfile

```diff
@@ -549,17 +548,46 @@ ENV["APP_STORE_STRINGS_FILE_NAME"]="AppStoreStrings.pot"
       download_path: "./fastlane/metadata"
     )
 
-    # Create the marketing and privacy URL files
-    metadata_locales.each { |locale|
-      marketing_path = Dir.pwd + "/metadata/#{locale}/marketing_url.txt"
-      write_file(marketing_path, 'https://woocommerce.com/mobile/')
-      
-      privacy_path = Dir.pwd + "/metadata/#{locale}/privacy_url.txt"
-      write_file(privacy_path, "https://automattic.com/privacy/")
+    # Ensure that localizations are present in the right places (some locale codes aren't correct)
+    {
+      "ar" => "ar-SA",
+      "de" => "de-DE",
+      "es" => "es-ES",
+      "fr" => "fr-FR",
+      "nl" => "nl-NL",
+      "zh-cn" => "zh-Hans",
+      "zh-tw" => "zh-Hant"
```

Comment:
Yeah, I very much agree that this is suboptimal 😅

I could refactor this, but IMHO this is something that our "fix up the localization pipeline" project should solve for us, and we should be able to just call a `download_translated_metadata` action or something and have it do the right thing. 

I'll be happy to burn this code at that time, but I'm not sure if it's worth interim work if we're gonna replace it anyway? 🤔

Reply:
TIL about our locale map helper project. GTK for when I'll be tackling the localization cleanup task!

Yeah as I said in my comment this YAML idea is very optional so given this project this can wait indeed